src/main/java/li/cil/scannable/data/ModRecipeProvider.java

```diff
@@ -8,55 +10,56 @@
-            .pattern("gqg")
-            .define('i', Tags.Items.INGOTS_IRON)
-            .define('b', Items.IRON_BARS)
-            .define('r', Tags.Items.DUSTS_REDSTONE)
-            .define('g', Tags.Items.INGOTS_GOLD)
-            .define('q', Tags.Items.GEMS_QUARTZ)
-            .group("scanner")
-            .unlockedBy("is_delving", LocationTrigger.TriggerInstance.located(LocationPredicate.inFeature(BuiltinStructures.MINESHAFT)))
-            .save(consumer);
+                .pattern("i i")
+                .pattern("brb")
+                .pattern("gqg")
+                .define('i', Items.IRON_INGOT)
+                .define('b', Items.IRON_BARS)
+                .define('r', Items.REDSTONE)
+                .define('g', Items.GOLD_INGOT)
+                .define('q', Items.QUARTZ)
+                .group("scanner")
+                .unlockedBy("is_delving", LocationTrigger.TriggerInstance.located(LocationPredicate.inFeature(BuiltinStructures.MINESHAFT)))
+                .save(consumer);
 
         ShapedRecipeBuilder.shaped(BLANK_MODULE.get())
-            .pattern("ggg")
-            .pattern("crc")
-            .pattern("cnc")
-            .define('g', Tags.Items.DYES_GREEN)
-            .define('c', Items.CLAY_BALL)
-            .define('r', Tags.Items.DUSTS_GLOWSTONE)
-            .define('n', Tags.Items.NUGGETS_GOLD)
-            .group("blank_module")
-            .unlockedBy("has_scanner", InventoryChangeTrigger.TriggerInstance.hasItems(SCANNER.get()))
-            .save(consumer);
+                .pattern("ggg")
+                .pattern("crc")
+                .pattern("cnc")
+                .define('g', Items.GREEN_DYE)
+                .define('c', Items.CLAY_BALL)
+                .define('r', Items.GLOWSTONE_DUST)
+                .define('n', Items.GOLD_NUGGET)
+                .group("blank_module")
+                .unlockedBy("has_scanner", InventoryChangeTrigger.TriggerInstance.hasItems(SCANNER.get()))
+                .save(consumer);
 
-        registerModule(RANGE_MODULE.get(), Tags.Items.ENDER_PEARLS).save(consumer);
+        registerModule(RANGE_MODULE.get(), Items.ENDER_PEARL).save(consumer);
         registerModule(ENTITY_MODULE.get(), Items.LEAD).save(consumer);
-        registerModule(FRIENDLY_ENTITY_MODULE.get(), Tags.Items.LEATHER).save(consumer);
-        registerModule(HOSTILE_ENTITY_MODULE.get(), Tags.Items.BONES).save(consumer);
-        registerModule(BLOCK_MODULE.get(), Tags.Items.STONE).save(consumer);
+        registerModule(FRIENDLY_ENTITY_MODULE.get(), Items.LEATHER).save(consumer);
+        registerModule(HOSTILE_ENTITY_MODULE.get(), Items.BONE).save(consumer);
+        registerModule(BLOCK_MODULE.get(), Items.STONE).save(consumer);
         registerModule(COMMON_ORES_MODULE.get(), Items.COAL).save(consumer);
-        registerModule(RARE_ORES_MODULE.get(), Tags.Items.GEMS_DIAMOND).save(consumer);
+        registerModule(RARE_ORES_MODULE.get(), Items.DIAMOND).save(consumer);
```

Comment:
No common tags for these in Fabric?

Reply:
The Fabric tag situation is a bit of a mess right now, in my opinion. They have some common tag *names*, but the API itself doesn't create or populate them. It's left up to mods to do that, even though it's redundant as soon as multiple are installed.

If you think it's worth adding data generation for the necessary common tags I'll do it. :)